Packs/Trellix_ePO/ModelingRules/Trellix_ePO/Trellix_ePO.xif

```diff
@@ -0,0 +1,71 @@
+    ThreatSeverity = to_string(parsed_fields -> ThreatSeverity),
+    AnalyzerHostName = parsed_fields -> AnalyzerHostName,
+    SourceMAC = uppercase(parsed_fields -> SourceMAC),
+    RawMACAddress = uppercase(parsed_fields -> RawMACAddress),
+    SourceProcessName = parsed_fields -> SourceProcessName
+| alter
+    xdm.alert.category = parsed_fields -> ThreatCategory,
+    xdm.alert.subcategory = parsed_fields -> ThreatType,
+    xdm.alert.name = parsed_fields -> ThreatName,
+    xdm.alert.original_threat_id = to_string(parsed_fields -> ThreatEventID),
+    xdm.alert.original_threat_name = parsed_fields -> ThreatName,
+    xdm.alert.severity = if(ThreatSeverity="1","critical",ThreatSeverity="2","high",ThreatSeverity="3","high",ThreatSeverity="4","medium",ThreatSeverity="5","low",ThreatSeverity="6","low",ThreatSeverity="7","informational",ThreatSeverity),
+    xdm.event.id = to_string(parsed_fields -> EventID),
+    xdm.event.is_completed = to_boolean(parsed_fields -> ThreatHandled),
+    xdm.network.rule = parsed_fields -> AnalyzerRuleName,
+    xdm.observer.action = parsed_fields -> ThreatActionTaken,
+    xdm.observer.content_version = parsed_fields -> AnalyzerContentVersion,
+    xdm.observer.name = coalesce(parsed_fields -> AnalyzerHostName,parsed_fields -> MachineName),
+    xdm.observer.type = parsed_fields -> AnalyzerName,
+    xdm.observer.unique_identifier = AgentGUID,
+    xdm.observer.version = parsed_fields -> AnalyzerVersion,
+    xdm.source.host.hostname = coalesce(parsed_fields ->SourceHostName, AnalyzerHostName),
+    xdm.source.host.mac_addresses = if(SourceMAC != null or RawMACAddress != null, arraydistinct(arraycreate(SourceMAC, RawMACAddress))),
+    xdm.source.host.os = OSName,
+    xdm.source.host.os_family = if(os contains "windows", XDM_CONST.OS_FAMILY_WINDOWS, os contains "mac", XDM_CONST.OS_FAMILY_MACOS, os contains "linux", XDM_CONST.OS_FAMILY_LINUX, os contains "android", XDM_CONST.OS_FAMILY_ANDROID, os contains "ios", XDM_CONST.OS_FAMILY_IOS, os contains "ubuntu", XDM_CONST.OS_FAMILY_UBUNTU, os contains "debian", XDM_CONST.OS_FAMILY_DEBIAN, os contains "fedora", XDM_CONST.OS_FAMILY_FEDORA, os contains "centos", XDM_CONST.OS_FAMILY_CENTOS, os contains "chrome", XDM_CONST.OS_FAMILY_CHROMEOS, os contains "solaris", XDM_CONST.OS_FAMILY_SOLARIS, os contains "scada", XDM_CONST.OS_FAMILY_SCADA),
+    xdm.source.interface = coalesce(SourceMAC,RawMACAddress),
+    xdm.source.ipv4 = agent_ipv4,
+    xdm.source.port = to_integer(parsed_fields -> SourcePort),
+    xdm.source.process.executable.directory = parsed_fields -> SourceFilePath,
+    xdm.source.process.executable.extension = arrayindex(regextract(SourceProcessName, "\.(.+)"), 0),
+    xdm.source.process.executable.filename = SourceProcessName,
+    xdm.source.process.executable.md5 = parsed_fields -> SourceProcessHash,
+    xdm.source.process.executable.path = concat(parsed_fields -> SourceFilePath, "\\", SourceProcessName),
```

Comment:
It seems like you used concat with double \\ instead of one, as mentioned during the demo.
@sdaniel6  
```suggestion
    xdm.source.process.executable.path = concat(parsed_fields -> SourceFilePath, "\", SourceProcessName),
```